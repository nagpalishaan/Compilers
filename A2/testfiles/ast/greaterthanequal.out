Program
  List
    Function
      IdDecl ID="n"
      Opt
        List
          DeclareVar
            IdDecl ID="a"
            List
          DeclareVar
            IdDecl ID="b"
            List
          AssignVar
            IdDecl ID="c"
            GreaterEqual
              IdUse ID="a"
              IdUse ID="b"
