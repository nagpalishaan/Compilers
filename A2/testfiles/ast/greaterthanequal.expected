Program
  List
    Function
      IdDecl ID="n"
      Opt
        List
          DeclareVar
            IdDecl ID="a"
            Opt
          DeclareVar
            IdDecl ID="b"
            Opt
          AssignVar
            IdDecl ID="c"
            GreaterEqual
              IdUse ID="a"
              IdUse ID="b"
